internal/v1/oscap.go

```diff
@@ -0,0 +1,109 @@
+package v1
+
+import (
+	"errors"
+)
+
+func OscapProfiles(distribution Distributions) (DistributionProfileResponse, error) {
+	switch distribution {
+	case Centos8:
+		fallthrough
+	case Rhel8:
+		fallthrough
+	case Rhel84:
+		fallthrough
+	case Rhel85:
+		fallthrough
+	case Rhel86:
+		fallthrough
+	case Rhel87:
+		fallthrough
+	case Rhel88:
+		fallthrough
+	case Rhel8Nightly:
+		return DistributionProfileResponse{
+			XccdfOrgSsgprojectContentProfileAnssiBp28Enhanced,
+			XccdfOrgSsgprojectContentProfileAnssiBp28High,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Intermediary,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Minimal,
+			XccdfOrgSsgprojectContentProfileCis,
+			XccdfOrgSsgprojectContentProfileCisServerL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL2,
+			XccdfOrgSsgprojectContentProfileCui,
+			XccdfOrgSsgprojectContentProfileE8,
+			XccdfOrgSsgprojectContentProfileHipaa,
+			XccdfOrgSsgprojectContentProfileIsmO,
+			XccdfOrgSsgprojectContentProfileOspp,
+			XccdfOrgSsgprojectContentProfilePciDss,
+			XccdfOrgSsgprojectContentProfileStig,
+			XccdfOrgSsgprojectContentProfileStigGui,
+		}, nil
+	case Centos9:
+		fallthrough
+	case Rhel9:
+		fallthrough
+	case Rhel91:
+		fallthrough
+	case Rhel92:
+		fallthrough
+	case Rhel9Nightly:
+		return DistributionProfileResponse{
+			XccdfOrgSsgprojectContentProfileAnssiBp28Enhanced,
+			XccdfOrgSsgprojectContentProfileAnssiBp28High,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Intermediary,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Minimal,
+			XccdfOrgSsgprojectContentProfileCis,
+			XccdfOrgSsgprojectContentProfileCisServerL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL2,
+			XccdfOrgSsgprojectContentProfileCui,
+			XccdfOrgSsgprojectContentProfileE8,
+			XccdfOrgSsgprojectContentProfileHipaa,
+			XccdfOrgSsgprojectContentProfileIsmO,
+			XccdfOrgSsgprojectContentProfileOspp,
+			XccdfOrgSsgprojectContentProfilePciDss,
+			XccdfOrgSsgprojectContentProfileStig,
+			XccdfOrgSsgprojectContentProfileStigGui,
+		}, nil
+	case Rhel90:
+		fallthrough
+	default:
+		return nil, errors.New("No profile for the specified distribution")
+	}
+}
+
+func OscapDistributionGeneralization(distribution Distributions) (string, error) {
```

Comment:
This method is added in one commit and removed in the next one. While this has no effect on the resulting merged code, it makes the history harder to read.